### PR TITLE
Add flow declaration for normalizr v3

### PR DESCRIPTION
**Pull request by re-gor**
Adapted copy of https://github.com/paularmstrong/normalizr/blob/master/index.d.ts

### CODE REVIEW

**Comment by re-gor** [comment]
As I see travis failed on flow v0.70, which is known issue due to https://github.com/flowtype/flow-typed/issues/2422

Am I did something wrong? I do not meet any issues in the local copy of repo. Help will be very appreciated

```
ERROR: normalizr_v2.x.x/flow_v0.25.x-
 * normalizr_v2.x.x/flow_v0.25.x- (flow-v0.70.0): Unexpected Flow errors(139):
   Segmentation fault
   
   null
```

**Comment by AndrewSouthpaw** [comment]
Nothing wrong, our flow CI is kinda borked... 😞

File: definitions/npm/normalizr_v3.x.x/flow_v0.25.x-/normalizr_v3.x.x.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+// @flow
+
```
Comment (AndrewSouthpaw) [inline definitions/npm/normalizr_v3.x.x/flow_v0.25.x-/normalizr_v3.x.x.js]: Please link to source `.ts` file for convenience and future referencing.

Comment (re-gor) [reply definitions/npm/normalizr_v3.x.x/flow_v0.25.x-/normalizr_v3.x.x.js]: Done. Added jsdoc comment with permanent link to normalizr's github repo

**Comment by villesau** [comment]
Nice!

**Comment by c960657** [comment]
Follow-up PR in #2739.

**Pull request merged**